docs/source/blocks.rst

```diff
@@ -0,0 +1,146 @@
+======
+Blocks
+======
+
+
+On a page, a block is a chunk of content which sits in a column.
+
+With Glitter, a block is a combination of a Django model and a template file,
+registered with the block admin so it can be selected in frontend templates.
+
+
+Quick example
+=============
+
+This example block defines a ``HeadlineText`` model, with just one attribute
+named ``text`` in ``models.py``.
+
+.. code:: python
+
+    from django.db import models
+
+    from glitter.models import BaseBlock
+
+
+    class HeadlineText(BaseBlock):
+        text = models.TextField()
+
+
+To make this usable in the frontend, we need to register this block in
+``blocks.py``.
+
+.. code:: python
+
+    from glitter.blockadmin import blocks
+
+    from .models import HeadlineText
+
+
+    @blocks.register(HeadlineText, category='Common')
+    class HeadlineTextAdmin(blocks.BlockAdmin):
```

Comment:
Don't think so.

Certain **models** are named `FooBlock`, and others are named `Foo`. I think we possibly wanted all models to be `FooBlock` in the past, but it was a bit annoying having to change the verbose_name every time.

However, now we've got blocks.py - maybe we should go through it again.